test/suite/test_checkpoint_snapshot02.py

```diff
@@ -66,6 +73,22 @@ def moresetup(self):
             self.nrows = 1000
             self.valuea = "aaaaa" * 100
 
+    def take_full_backup(self, fromdir, todir):
+        # Open up the backup cursor, and copy the files.  Do a full backup.
+        cursor = self.session.open_cursor('backup:', None, None)
+        self.pr('Full backup from '+ fromdir + ' to ' + todir + ': ')
+        os.mkdir(todir)
+        while True:
+            ret = cursor.next()
+            if ret != 0:
+                break
+            bkup_file = cursor.get_key()
+            sz = os.path.getsize(fromdir + "/" + bkup_file)
+            self.pr('Copy from: ' + bkup_file + ' (' + str(sz) + ') to ' + todir)
+            shutil.copy(fromdir + "/" + bkup_file, todir)
```

Comment:
It's better to use `os.path.join()` instead of concatenating the file path manually.
Also, we do it twice on lines 86 and 88. It's would be better if we avoid duplication.

Reply:
Fixed. Thanks.